src/server/apis/webhook.ts

```diff
@@ -38,7 +41,87 @@ function getRouter(): express.Router {
     }
   });
 
+  webhookRouter.post('/:action', async (request: express.Request, response: express.Response) => {
+    const loginDetails = await getLoginFromRequest(github, request);
+    if (!loginDetails) {
+        response.sendStatus(400);
+        return;
+    }
+
+    if (!request.body.org) {
+      response.status(400).send('No org provided.');
+      return;
+    }
+
+    if (request.params.action === 'add') {
+      const githubAPIResponse = await fetch(
```

Comment:
Can we refactor this into github.ts? There's a v3 `get` API call, so a similar one can be added for `post`.  This will factor out some of the boilerplate like setting tokens, user agent, headers etc.

Reply:
Added - didn't spot that before.